smdebug/pytorch/hook.py

```diff
@@ -359,6 +360,7 @@ def forward_pre_hook(self, module, inputs):
         )
 
         self.profiler_config_parser.load_config()
+        ConfigValidator.validate_profiler_config(self.profiler_config_parser)
```

Comment:
Wouldn't we have to do the same for TensorFlow too?

My suggestion - move this call to inside `load_config` so that it's framework agnostic (and we don't need to track where `load_config` is called).

Reply:
Done.